### ISSUE TITLE
css/pcss import issues with vite > 2.7.0

### ISSUE THREAD

**Issue by joepagan** [nystudio107/craft-vite#18]
### Describe the bug

Nesting a css import in a "top-level" css/pcss file loaded by an entry point fails in 2.7.0, this works as expected in 2.6.14

### To reproduce

`home.ts` entry point:

```.ts
import '@/css/app.pcss';
```

`app.pcss`:

```
@import 'tailwindcss/base';
@import 'tailwindcss/components';
@import './components/index.pcss';
@import './overrides/index.pcss';
@import 'tailwindcss/utilities';
@import './pages/index.pcss';
@import 'vendor.pcss';
```

loads the app.pcss in network tab with 200 status, however in the js console:

> Failed to load module script: Expected a JavaScript module script but the server responded with a MIME type of "". Strict MIME type checking is enforced for module scripts per HTML spec.

As we discussed in discord, I tried changing these files to use the `.css` extension, however this resulted in:

>Failed to load module script: Expected a JavaScript module script but the server responded with a MIME type of "text/css". Strict MIME type checking is enforced for module scripts per HTML spec.

### Expected behaviour

no js error logged & styles to load

### Versions
- Plugin version: "nystudio107/craft-vite": "1.0.16",
- Craft version: "craftcms/cms": "3.7.13",
- vite: > 2.7.0

**Comment by khalwat** [comment]
This is not an issue with the Craft Vite plugin --  I'd suggest filing it over here: https://github.com/vitejs/vite/issues

**Comment by joepagan** [comment]
Just going to follow this up, I think I have resolved my issue.

Essentially I was trying to keep the (what used to be a somewhat "industry standard") `@` as a root to all the app's files, but, `vite:^2.7.0` wants any imports to start with a prefixed `/` so I had tried to change my alias to `/@` keeping that somewhat but it looks like that caused these issues reported specifically. weird.

Instead, I had to replace all my paths throughout entry points & components that were using `@` path prefix to instead use relative paths instead.

Not sure I like it but that's seemingly the only way 🤷

**Comment by khalwat** [comment]
That doesn't sound right... you should be able to set up `@` as an alias, and uses that as a prefix

**Comment by joepagan** [comment]
As it didn't seem like a true fix I have been on vite:2.6.14 for a while with this app.

I know this is many moons later but hopefully will help someone, I'll no doubt visit this again in future. 

I got this working with latest vite:2.9.

My original project I was posting about was originally based off your https://github.com/nystudio107/craft project roughly 12 months ago, I made a new project using the same project and compared the resulting files.

I strongly suspect it was due to the modifications in my tsconfig.json, potentially related to:

```diff
-    "baseUrl": ".",
+    "baseUrl": "node_modules",
```

The above potentially relates to https://github.com/vitejs/vite/issues/8429#issuecomment-1144458483 though not sure how directly linked the tsconfig.json and the vite config are.

Of course could be a bunch of new version of npm packages now being used could make a difference too.
